apps/ios/Shared/Views/Chat/VerifyCodeView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  VerifyCodeView.swift
+//  SimpleX (iOS)
+//
+//  Created by Evgeny on 10/12/2022.
+//  Copyright © 2022 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+
+struct VerifyCodeView: View {
+    @Environment(\.dismiss) var dismiss: DismissAction
+    var displayName: String
+    @State var connectionCode: String?
+    @State var connectionVerified: Bool
+    var verify: (String?) -> (Bool, String)?
+    @State private var showCodeError = false
+
+    var body: some View {
+        if let code = connectionCode {
+            verifyCodeView(code)
+        }
+    }
+
+    private func verifyCodeView(_ code: String) -> some View {
+        ScrollView {
+            let splitCode = splitToParts(code, length: 24)
+            VStack(alignment: .leading) {
+                Group {
+                    HStack {
+                        if connectionVerified {
+                            Image(systemName: "checkmark.shield")
+                                .foregroundColor(.secondary)
+                            Text("\(displayName) is verified")
+                        } else {
+                            Text("\(displayName) is not verified")
+                        }
+                    }
+                    .frame(height: 24)
+
+                    QRCode(uri: code)
+                        .padding(.horizontal)
+
+                    Text(splitCode)
+                        .multilineTextAlignment(.leading)
+                        .font(.body.monospaced())
+                        .lineLimit(20)
+                        .padding(.bottom, 8)
+                }
+                .frame(maxWidth: .infinity, alignment: .center)
+
+                Text("To verify end-to-end encryption with your contact compare (or scan) the code on your devices.")
```

Comment:
parameterize view with text so it says "group member" in place of "your contact" in groups?

Reply:
full sentence not just subject

Reply:
Don’t think it’s important

Reply:
Nobody will verify group members I think…